norse/torch/module/snn_cg.py

```diff
@@ -0,0 +1,81 @@
+import torch
+from typing import Any, Callable, List, Optional, Tuple
+
+from norse.torch.functional.lif import LIFFeedForwardState
+from norse.torch.module.snn import SNN, FeedforwardActivation
+
+# ===================================================================
+# Implement various SNN cells using CUDA Graphs
+# <https://pytorch.org/blog/accelerating-pytorch-with-cuda-graphs/>
+# ===================================================================
+
+class SNNCellCG(SNN):
```

Comment:
This should by the way inherit from `SNNCell`, right?

Reply:
Absolutely! My bad...